UC-Berkeley-Extension/global
# Drupal components styleguide

## Asana Ticket(s)

## Description
- Adds Drupal components scss
- Fixes font paths so it applies on Drupal
- File structure reorg to remove errors on `npm start`
- Moved "Tabs block to above page title, and they should be rendering as tabs instead of list items.
<img width="576" alt="Screen Shot 2020-09-25 at 3 49 27 PM" src="https://user-images.githubusercontent.com/10488517/94314699-b5233b00-ff46-11ea-84ce-43469e0269ce.png">

## Testing instructions
- `ddev start`
- (on a separate terminal window) Navigate to `web/themes/custom/bglobal`, `npm start`. Confirm that the yellow error lines that were present before are no longer there.
- Open a local Drupal page. Check that correct fonts are applying.
- Check that the admin actions tabs are rendering as a horizontal line of tabs.
